manifests/s/SimAirIo/SimAirClient/0.0.7-beta/SimAirIo.SimAirClient.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SimAirIo.SimAirClient
+PackageVersion: 0.0.7-beta

```

Comment:
```suggestion
PackageVersion: 1.0.0
```

Reply:
The download URL suggests it's not really version 1.0.0, though. I guess they do not set the version number in the installer correctly, i.e. the installer will always have the default of 1.0.0.

Reply:
Should we still set this version number in the manifest?